junit-engine-api/src/main/java/org/junit/gen5/engine/ExecutionRequest.java

```diff
@@ -25,12 +25,16 @@
 
 	private final EngineExecutionListener engineExecutionListener;
 
+	private ConfigurationParameters configurationParameters;
+
 	private final Map<String, Object> attributes = new HashMap<>();
```

Comment:
I think `configurationParameters` should replace `attributes` or do you want to do this in a later step?